Library/Formula/vc4asm.rb

```diff
@@ -0,0 +1,32 @@
+class Vc4asm < Formula
+  desc "Macro assembler for Broadcom VideoCore IV aka Raspberry Pi GPU"
+  homepage "http://maazl.de/project/vc4asm/doc/index.html"
+  url "https://github.com/maazl/vc4asm/archive/V0.1.7.tar.gz"
+  sha256 "db9c50c1cc035a183ce8305a82a01cad08a246d13c718c420a8762296b00e3de"
+
+  depends_on "cmake" => :build
+
```

Comment:
Can you add a `needs :cxx11` on line 9 and push the patch down a couple of lines after, Thanks!